lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -22,26 +22,27 @@ def initialize(vmpooler_hosts, options)
       @logger = options[:logger]
       @hosts = vmpooler_hosts
       @credentials = load_credentials(@options[:dot_fog])
+
     end
 
     def load_credentials(dot_fog = '.fog')
       creds = {}
-
       if fog = read_fog_file(dot_fog)
         if fog[:default] && fog[:default][:vmpooler_token]
           creds[:vmpooler_token] = fog[:default][:vmpooler_token]
         else
           @logger.warn "Credentials file (#{dot_fog}) is missing a :default section with a :vmpooler_token value; proceeding without authentication"
         end
       else
-        @logger.warn "Credentials file (#{dot_fog}) is empty; proceeding without authentication"
+        @logger.warn "Credentials file (#{dot_fog}) is missing a :default section; proceeding without authentication"
```

Comment:
@Sukanyade2014 isn't it the case that when this happens, it can be more than just that you're missing a defaults section?